src/app/framework-modules/elements/ct-field-generator/component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, Input, Output, EventEmitter} from '@angular/core';
+import {FormBuilder, FormGroup, Validators, FormControl} from '@angular/forms';
+import { Field, Form } from '../../formdata/form/form';
+import { FormData } from '../../formdata/form/formData';
+
+
+
+@Component({
+	selector: 'ct-field-generator',
+	templateUrl: './component.html',
+	styleUrls: ['./component.scss']
+})
+
+export class CTFieldGen {

```

Comment:
Would it make sense to add a javadoc here to explain what this does?  Also, what does the "CT" prefix mean in "CTFieldGen"?  Finally, what is "Gen"?  Generator?  Maybe write it out in full, so it's more self documenting?